lib/aws/bootstrap/tf-default-vars.j2.tf

```diff
@@ -78,7 +78,7 @@ variable "eks_cluster_name" {
 
 variable "eks_access_cidr_blocks" {
   description = "Kubernetes cluster name"
-  default     = "{{ eks_access_cidr_blocks }}"
+  default     = {{ eks_access_cidr_blocks }}
```

Comment:
@deimosfr no needs to double-quote this?

Reply:
Nope, because it's a list of string which is expected